docs/starters.yml

```diff
@@ -1075,7 +1084,7 @@
   description: Blog Template X Contentful, Twitter and Facebook style
   tags:
     - Blog
-    - Tags
```

Comment:
I think intention here is that starters has support for post tags?

Reply:
Mhhh, no one used that even though the starter has it. You could also call that categories, so what to choose here then?

Reply:
I probably miss-understood intention (I thought this was one of invalid tags). On that I have no opinion